core/events/bus.go

```diff
@@ -141,6 +141,8 @@ const (
 	ExpiredOrdersEvent
 	DistressedPositionsEvent
 	StopOrderEvent
+	FundingPeriodEvent
+	FundingPeriodDataPointEvent
```

Comment:
also, do we need to add it to the gql model in gqlgen.yml?

Reply:
Probably, but I think we're slowly creeping into this ticket really:
https://github.com/vegaprotocol/vega/issues/8763

Reply:
yeah, sure it doesn't need to be in this PR.